doc/source/whatsnew/v1.4.0.rst

```diff
@@ -93,6 +93,23 @@ Multithreaded CSV reading with a new CSV Engine based on pyarrow
 :func:`pandas.read_csv` now accepts ``engine="pyarrow"`` (requires at least ``pyarrow`` 0.17.0) as an argument, allowing for faster csv parsing on multicore machines
 with pyarrow installed. See the :doc:`I/O docs </user_guide/io>` for more info. (:issue:`23697`)
 
+.. _whatsnew_140.enhancements.redis_io:
+
+Redis IO Functionality
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We've added Redis to our supported IO capabilities. This allows one to serialize DataFrame obects to and from Redis, while maintaining datatypes. (:issue:`43415`) 
+
+.. code-block:: python 
+
+    import pandas as pd 
+    import redis
+
+    df = pd.DataFrame(data=[1.2.3], columns=['A'. 'B', 'C'])
+    
+    redis_conn = redis.StrictRedis(host="your host", port=6379, db=0)
+
+    df.redis.to_redis(redis_conn, alias="test")
```

Comment:
wouldnt it just be `df.to_redis`?

Reply:
Good catch, that was a leftover from moving from an accessor extension. Resolved in commit a4704b8dc38353c2e7891c1e9fc7701053e9490f